app/controllers/nodes_controller.rb

```diff
@@ -16,6 +18,18 @@ def show
     @minion = Minion.find(params[:id])
   end
 
+  def bootstrap
+    salt = Pharos::Salt.new
+    available_roles = %i(master)
+    salt.minions.keys.each do |minion|
```

Comment:
I think I'm done with the other one. I'll check and I'll remove the [WIP] flag. Maybe we should merge that first and update this code.